site/source/partials/header.hbs

```diff
@@ -27,14 +27,41 @@
   <!-- stylesheet -->
   <link rel="stylesheet" href="{{assets}}css/style.css">
 
-  <!-- Cedar -->
-
-  <!-- esri Cedar -->
-  <script type="text/javascript" src="http://square.github.io/crossfilter/d3.v3.min.js"></script>
-  <script type="text/javascript" src="{{assets}}js/vega.js"></script>
```

Comment:
No. I'm not sure of the history of why the docs site has been pointing to [a locally hosted copy of vega.js](https://github.com/Esri/cedar/blob/master/js/vega.js), but now that I've tested the whole site on CDN hosted versions of vega and we know that the whole site works on v1.5 (except the pie chart examples which require v1.4.3) I think we can get rid of that file and just use the CDN for the site and bower for the builds.